blog/en/i-built-a-deep-research-with-open-source-so-can-you.md

```diff
@@ -87,6 +87,9 @@ Here is an example reasoning trace as DeepSeek refines the question “How has t
 
 ```bash
 <think>
+```
+
+```
```

Comment:
@septemberfd will there be a visable gap between the think text and the text below? It should be in a single box without any coloring

Reply:
yes it will be a separate box same as the output below